app/assets/v2/css/base.css

```diff
@@ -1671,3 +1676,20 @@ input.is-invalid {
     display: none;
   }
 }
+
+/* TODO : Remove after GDPR */
+.notify_policy_update {
+  font-family: "Muli", sans-serif;
+}
+
+.notify_policy_update h2 {
+  letter-spacing: 0;
+}
+
+.modal.notify_policy_update {
+  max-width: 34rem;
+}
+
+.notify_policy_update .button {
+  padding: 4px 20px;
+}
```

Comment:
Can you add a newline at the end? :sweat_smile:

Reply:
As in below the button ?